### PR TITLE
Upgrade zinc and nailgun to latest

### PR DESCRIPTION
**Pull request by jvican**

### CODE REVIEW

**Comment by jvican** [comment]
Failing because we're waiting on Maven Central to sync up the published artifacts.

**Comment by jvican** [comment]
test performance plase

**Comment by jvican** [comment]
test performance please

**Comment by bloopoid** [comment]
performance test scheduled: 1 job(s) in queue, 0 running.

**Comment by bloopoid** [comment]
Performance test finished successfully.

Benchmarks is based on merging with master

**Comment by Duhemm** [comment]
test performance please

**Comment by bloopoid** [comment]
performance test scheduled: 1 job(s) in queue, 0 running.

**Comment by jvican** [comment]
I just removed the community build label so that we don't run it again, but it completed successfully in https://ci.scala-lang.org/scalacenter/bloop/3114/11, which is great news.

File: bin/run-benchmarks.sh [hunk 1]
```diff
@@ -36,9 +37,8 @@ main() {
     mkdir -p "$BLOOP_HOME"
 
     JMH_CMD="$BLOOP_JMH_RUNNER"
-    LOG_FILE="$BLOOP_HOME/$(date +"%F-%H%M%S")-benchmarks-log.txt"
```
Comment (Duhemm) [inline bin/run-benchmarks.sh]: Can you hoist that value up so that `LOG_FILE` has a default value?

Comment (jvican) [reply bin/run-benchmarks.sh]: What is the value of doing that? We check against the existence of `LOG_FILE` in the trap exit handler.

Comment (Duhemm) [reply bin/run-benchmarks.sh]: But I don't want the script to crash if I didn't give it a `--logfile`

Comment (jvican) [reply bin/run-benchmarks.sh]: I do, I really wanna make sure that both the listener and this script are on-sync. Do you run this manually?

**Comment by jvican** [comment]
For the record, Bloop does no longer require the build to have coursier enabled globally.

**Pull request merged**